packages/lib/src/components/PayPal/components/PaypalComponent.tsx

```diff
@@ -32,10 +32,6 @@ export default function PaypalComponent(props: PayPalComponentProps) {
         } else {
             script.load().then(handlePaypalLoad);
         }
-
-        return () => {
```

Comment:
No longer needed?

Reply:
No, this was wrongly removed. I rolled back the change, thanks for the heads up.